src/modules/auth/actions/set-login-account.js

```diff
@@ -1,10 +1,12 @@
 import { useUnlockedAccount } from 'modules/auth/actions/use-unlocked-account'
+import { loadAccountTrades } from 'modules/my-positions/actions/load-account-trades'
 import logError from 'utils/log-error'
 
 // If there is an available unlocked account, set as the user's sending address.
 export const setLoginAccount = (autoLogin, account, callback = logError) => (dispatch, getState) => {
   // If autoLogin=true, use an unlocked local Ethereum node (if present)
   if (autoLogin) {
```

Comment:
Indirectly relevant to this PR, but, it appears the only callsite for `setLoginAccount` is app/actions/init-augur.js:
```javascript
dispatch(setLoginAccount(true, account))
```
Unless I'm misunderstanding something, the first argument should be determined by the user-specified `env.autoLogin` setting, rather than being hardcoded to `true`.

Reply:
Not comfortable changing it. Not sure the difference between meta mask and other login methods. If I change to use auto-login from env then it doesn't log in even though meta mask is logged in. 

I'll leave to @stephensprinkle and make a clean up task in CH.

https://app.clubhouse.io/augur/story/6582/cleanup-initaugur-takes-true-for-auto-login